src/omnicore/doc/release-notes.md

```diff
@@ -1 +1,236 @@
+=================
+
+v0.0.10 is a major release and consensus critical in terms of the Omni Layer protocol rules. An upgrade is mandatory, and highly recommended.
+
+Please report bugs using the issue tracker on GitHub:
+
+  https://github.com/OmniLayer/omnicore/issues
+
+Upgrading and downgrading
+=========================
+
+How to upgrade
+--------------
+
+If you are running an older version, shut it down. Wait until it has completely shut down, then copy the new version of `omnicored`, `omnicore-cli` and `omnicore-qt`, if applicable.
+
+During the first startup historical Omni transactions are reprocessed and Omni Core will not be usable for approximately 15 minutes up to two hours. The progress of the initial scan is reported on the console, the GUI and written to the `debug.log`. The scan may be interrupted, but can not be resumed, and then needs to start from the beginning.
+
+Downgrading
+-----------
+
+Downgrading is not supported as older versions will not provide accurate information.
+
+Compatibility
+-----------
+
+Omni Core can be used as replacement for Bitcoin Core 0.10.0, 0.10.1, 0.10.2 and 0.10.3, and switching between Omni Core and Bitcoin Core is fully supported.
+
+Imported changes and notes
+==========================
+
+Upgrade to Bitcoin Core 0.10.3
+------------------------------
+
+The underlying base of Omni Core was upgraded to Bitcoin Core 0.10.3.
+
+Please see the following release notes for further details:
+
+- https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.10.0.md
+- https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.10.1.md
+- https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.10.2.md
+- https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.10.3.md
+
+Headers-first synchronization
+-----------------------------
+
+With the upgrade to Bitcoin Core 0.10 headers-first synchronization is supported and blocks are downloaded in parallel.
+
+Blocks are no longer stored in order on disk, and as result, the block files and databases are not backwards-compatible with older versions of Bitcoin Core prior 0.10.
+
+Dust threshold values
+---------------------
+
+The default `"minrelaytxfee"` was raised from `0.00001 BTC` to `0.00005 BTC`in Bitcoin Core 0.10.3 as temporary measure against massive memory pool bloat.
+
+This has an impact on the output values of Omni transactions, which are chosen to be as low as possible. As per default, Omni transactions created with Master Core had output values between `0.00000546 BTC` and `0.00000882 BTC`, whereby the new output values are between `0.0000273 BTC` and `0.0000441 BTC`.
+
+To continue to create transactions with lower values, start Omni Core with `-minrelaytxfee=0.00001` or add the following to your `bitcoin.conf`:
+```
+minrelaytxfee=0.00001
+```
+
+A lower `"minrelaytxfee"` may result in higher memory consumption, but is usually safe to use.
+
```

Comment:
Shouldn't we note here that using a lower ```minrelaytxfee``` than default may cause difficulty getting the transaction mined (especially if you're only connected to nodes with the new higher default)?